cognite/neat/_client/_api/data_modeling_loaders.py

```diff
@@ -736,6 +792,32 @@ def get_id(cls, item: DataModel | DataModelApply | DataModelId | dict) -> DataMo
             return DataModelId.load(item)
         return item
 
+    @classmethod
+    def merge(cls, local: DataModelApply, remote: DataModel) -> DataModelApply:
+        if local.as_id() != remote.as_id():
+            raise ValueError(f"Cannot merge data models with different IDs: {local.as_id()} and {remote.as_id()}")
+        existing_view = {
+            view.as_id() if isinstance(view, View) else view: view.as_write() if isinstance(view, View) else view
```

Comment:
Could we maybe have a type hint here? existing_view might have different types depending on "view